src/sage/coding/binary_code.pyx

```diff
@@ -3368,14 +3373,16 @@ cdef class BinaryCodeClassifier:
                 e[k] = 0 # see state 12 and 17
                 state = 2 # continue down the tree
 
-            elif state == 5: # same as state 3, but in the case where we haven't yet defined zeta
-                             # i.e. this is our first time down the tree. Once we get to the bottom,
-                             # we will have zeta = nu = rho, so we do:
+            elif state == 5:
+                # same as state 3, but in the case where we haven't yet defined zeta
+                # i.e. this is our first time down the tree. Once we get to the bottom,
+                # we will have zeta = nu = rho, so we do:
                 zf__Lambda_zeta[k] = Lambda[k]
                 zb__Lambda_rho[k] = Lambda[k]
                 state = 4
 
-            elif state == 6: # at this stage, there is no reason to continue downward, so backtrack
+            elif state == 6:
+                # at this stage, there is no reason to continue downward, so backtrack
                 j = k
```

Comment:
I don't know the code, but adding the comment after `sate = 6:` seems intentional. Respecting the intention seems more important than PEP8 in this case.